HTTPRequest.cs

```diff
@@ -5,120 +5,154 @@
 
 namespace DNWS
 {
-  public class HTTPRequest
```

Comment:
Don't change my code layout, e.g., the tab size, otherwise, there will be a lot of code changes and I can't find your code easily.